### ISSUE TITLE
Missing helper when function defined in gulpfile.

### ISSUE THREAD

**Issue by andeersg** [shannonmoeller/gulp-hb#21]
Hi,

I have a problem when I try to define a helper inside the gulpfile:
```JS
gulp.task('test', function () {
    return gulp
        .src('pages/*.hbs')
        .pipe(hb({
            data: 'testing/*.{js,json}',
            helpers: {
              lower: function (text) {
                return String(text).toLowerCase();
              }
            },
            partials: 'partials/**/*.hbs'
        }))
        .pipe(extname())
        .pipe(gulp.dest('dist2/'));
});
```

If I run this I get the following error: ``Error: Missing helper: "lower"``.

Both partials and data works.

**Comment by shannonmoeller** [comment]
Well that's no good. Looks like there's an error in my tests that's allowing bleed-through from one test to another. I'm on it. Thanks!

**Comment by shannonmoeller** [comment]
Fixed and released as v2.6.0.

**Comment by andeersg** [comment]
Awesome :)
